universal_notifications/models.py

```diff
@@ -64,7 +64,12 @@ def send_message(self, message, **data):
             return False
 
         if not isinstance(message, six.string_types):
-            message = unicode(message)
+            try:
```

Comment:
six.u will return string only if the message passed is a string. In case a non-string type is passed (see the if), six.u on Py3 will return the same value. https://repl.it/G0uU/0